addons/website_sale/views/website_sale_menus.xml

```diff
@@ -65,11 +65,11 @@
         <menuitem id="menu_ecommerce_payment_providers"
             name="Payment Providers"
             action="payment.action_payment_provider"/>
-        <menuitem id="menu_ecommerce_payment_tokens"
-            action="payment.action_payment_token"
-            groups="base.group_no_one"/>
         <menuitem id="menu_ecommerce_payment_methods"
             action="payment.action_payment_method"
+            name="Payment Methods"/>
+        <menuitem id="menu_ecommerce_payment_tokens"
+            action="payment.action_payment_token"
```

Comment:
You'll have to add a sequence to the menus if you want existing databases to have the methods menu before the tokens one (since they are already created and have both the same sequence, they'll keep the ordering by id they had before)

Reply:
good catch